contrib/init/openrc/containerd.initd

```diff
@@ -0,0 +1,26 @@
+#!/sbin/openrc-run
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+description="Containerd container runtime"
```

Comment:
I'm not sure if it makes sense to maintain an openrc init script for containerd, as it's a separate project, and we don't ship it as part of the `docker-ce` package itself.

As to "how containerd is started";

- for most distros (where systemd is used as init), the docker systemd unit depends on the containerd service; the dockerd daemon is configured with the `--container` option, which defines the path to containerd's socket to connect with https://github.com/moby/moby/blob/219374e2cd1b516434ccbe9980ae42a4611bcbcc/contrib/init/systemd/docker.service#L13
- if the `--containerd` flag is not set, then;
    - `dockerd` will check if containerd's socket is present
    - if not, it will start containerd as a child process 


The last scenario is mostly to allow the `dockerd` process to be run manually (`sudo dockerd`) for debugging purposes. The "detect if containerd is already running" scenario is known to be potentially "racy" in a setup where containerd is actually running as a separate service.